scanners/ios/icon.go

```diff
@@ -85,11 +94,11 @@ func lookupIconByTarget(projectPath string, target xcodeproj.Target, basepath st
 	return icons, nil
 }
 
-func mainTargetOfScheme(proj xcodeproj.XcodeProj, scheme string) (xcodeproj.Target, error) {
+func mainTargetOfScheme(proj xcodeproj.XcodeProj, scheme string) (xcodeproj.Target, bool, error) {
 	projTargets := proj.Proj.Targets
 	sch, ok := proj.Scheme(scheme)
 	if !ok {
-		return xcodeproj.Target{}, fmt.Errorf("Failed to find scheme (%s) in project", scheme)
+		return xcodeproj.Target{}, false, fmt.Errorf("Failed to find scheme (%s) in project", scheme)
```

Comment:
i think the only one possible error just codes that the function was not able to find the scheme, what's the point of returning both a bool and an error with this information